### PR TITLE
Empty search returns all schools

### PR DESCRIPTION
**Pull request by stormwild**

### CODE REVIEW

File: routes/api/v1/schools.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ router.post('/', auth.optional, schools_controller.createSchool);
 
 router.get('/', auth.optional, schools_controller.getAllSchoolsWithPagination);
 
-router.get('/:id', auth.optional, schools_controller.getSchoolById);
+router.get('/search/:term?', auth.optional, schools_controller.searchSchool);
```
Comment (bhansaliraunak) [inline routes/api/v1/schools.js:10]: Need to change from get to post
```suggestion
router.post('/search/:term?', auth.optional, schools_controller.searchSchool);
```

Comment (bhansaliraunak) [reply routes/api/v1/schools.js:10]: @stormwild FYI

Comment (stormwild) [inline routes/api/v1/schools.js:10]: This is an optional route param.

Comment (bhansaliraunak) [reply routes/api/v1/schools.js:10]: @stormwild 
Search functionality isn't working with http get method. 
However, the same's working with post method

Comment (bhansaliraunak) [reply routes/api/v1/schools.js:10]: @stormwild

**Review by bhansaliraunak — APPROVED** [review]
Approving changes since @stormwild 's search functionality is working with HTTP Get method.

**Pull request merged**